domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -1158,67 +1176,47 @@ private void doDelete(SQLConnection connection, boolean transactionMode, String
       String fieldName, String where, boolean returnCount, boolean returnIdField, boolean setId,
       List<FacetField> facets, Handler<AsyncResult<Results<T>>> replyHandler) {
```

Comment:
There should be more unit test coverage of doGet. You may drop "private" to make this function accessible for unit tests.
Unit tests help code review because they prove that the function works for basic input and for edge cases.

Reply:
Dropping "private" results in no explicit modifier and is package-private: https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html
Only classes in the same package can access it, including unit test classes in that package.